server/command/debug.go

```diff
@@ -4,55 +4,95 @@
 package command
 
 import (
+	"fmt"
+
 	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/pkg/errors"
 
-	"github.com/mattermost/mattermost-plugin-apps/server/apps"
-	"github.com/mattermost/mattermost-plugin-apps/server/http/helloapp"
+	"github.com/mattermost/mattermost-plugin-apps/server/api"
+	"github.com/mattermost/mattermost-plugin-apps/server/examples/hello"
+	"github.com/mattermost/mattermost-plugin-apps/server/examples/hello/builtin_hello"
+	"github.com/mattermost/mattermost-plugin-apps/server/examples/hello/http_hello"
+	"github.com/mattermost/mattermost-plugin-apps/server/http/dialog"
 	"github.com/mattermost/mattermost-plugin-apps/server/utils/md"
 )
 
 func (s *service) executeDebugClean(params *params) (*model.CommandResponse, error) {
-	_ = s.apps.Mattermost.KV.DeleteAll()
-	return normalOut(params, md.MD("Deleted all KV records"), nil)
+	_ = s.api.Mattermost.KV.DeleteAll()
+	_ = s.api.Configurator.StoreConfig(&api.StoredConfig{})
+	return normalOut(params, md.MD("Deleted all KV records and emptied the config."), nil)
 }
 
 func (s *service) executeDebugBindings(params *params) (*model.CommandResponse, error) {
-	bindings, err := s.apps.API.GetBindings(&apps.Context{
-		ActingUserID: params.commandArgs.UserId,
-		UserID:       params.commandArgs.UserId,
-		TeamID:       params.commandArgs.TeamId,
-		ChannelID:    params.commandArgs.ChannelId,
-	})
+	bindings, err := s.api.Proxy.GetBindings(api.NewCommandContext(params.commandArgs))
 	if err != nil {
 		return normalOut(params, md.MD("error"), err)
 	}
 	return normalOut(params, md.JSONBlock(bindings), nil)
 }
 
-func (s *service) executeDebugEmbedded(params *params) (*model.CommandResponse, error) {
-	_, err := s.apps.API.Call(&apps.Call{
-		URL: s.apps.Configurator.GetConfig().PluginURL + apps.HelloAppPath + helloapp.PathSendSurvey,
-		Context: &apps.Context{
-			AppID:        helloapp.AppID,
-			ActingUserID: params.commandArgs.UserId,
-			ChannelID:    params.commandArgs.ChannelId,
-			TeamID:       params.commandArgs.TeamId,
-			UserID:       params.commandArgs.UserId,
-		},
-	})
+func (s *service) executeDebugEmbeddedForm(params *params) (*model.CommandResponse, error) {
+	cc := api.NewCommandContext(params.commandArgs)
```

Comment:
@larkox I totally bastardized the Embedded functionality to make this PR work, will appreciate a quick discussion on how we can restore the functionality, it's pretty essential to `Hello` app :)